hassio/homeassistant.py

```diff
@@ -452,21 +456,35 @@ def check_port():
                 pass
             return False
 
-        while time.monotonic() - start_time < self.wait_boot:
+        while True:
+            await asyncio.sleep(10)
+
+            # Check if Container is is_running
+            if not await self.instance.is_running():
+                _LOGGER.error("HomeAssistant is crashed!")
+                break
+
+            # Running DB Migration
+            if migration_file.exists():
+                if not migration_progress:
+                    migration_progress = True
+                    _LOGGER.info("HomeAssistant record migration in progress")
+                continue
+            elif migration_progress:
+                migration_progress = False
+                start_time = time.monotonic()
+                _LOGGER.info("HomeAssistant record migration done")
```

Comment:
We should only log this if migration_process was not True.

Reply:
that will be on line 473